src/CMakeLists.txt

```diff
@@ -507,7 +507,8 @@ if(NOT WITH_SYSTEM_BOOST)
 endif()
 
 if(HAVE_QATZIP)
-  list(APPEND ceph_common_deps ${qatzip_LIBRARIES})
+  # TODO: only the compression plugins should depend on QAT
+  list(APPEND ceph_common_deps QAT::qat QAT::usdm QAT::zip)
```

Comment:
i did this as a separate https://github.com/ceph/ceph/pull/54653 since it doesn't depend on anything here